src/Entity/Mooc/Quizz.php

```diff
@@ -16,30 +16,31 @@ class Quizz extends BaseMoocElement
     /**
      * @var string
      *
-     * @ORM\Column(type="text")
+     * @ORM\Column
      *
      * @Assert\NotBlank
+     * @Assert\Url
      */
-    private $typeForm;
+    private $typeFormUrl;
```

Comment:
Not a big deal but we can name it `$typeformUrl` without CamelCase for Typeform.